packages/amesos2/src/Amesos2_EpetraMultiVecAdapter_def.hpp

```diff
@@ -135,6 +135,14 @@ bool MultiVecAdapter<Epetra_MultiVector>::isConstantStride() const
 }
 
 
+bool MultiVecAdapter<Epetra_MultiVector>::isGIDContiguous() const
+{
+  // Epetra_MultiVector - have a method to retrieve map???
+  return true; // temp for now...
```

Comment:
Not used, was added and used before I updated the solver parameter list to pass IsContiguous setting; will remove